force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +42,7 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        return age >= 18 ? true : false;
```

Comment:
You can actually simplify this further. 

Does age >= 18 already result in a boolean we can return?

Reply:
(assuming we don't need to null check)